backend/api/views.py

```
@@ -1,96 +1,75 @@
-)
-from .serializers import (
-    RoleSerializer, CustomUserSerializer, CompanySerializer, PaymentSerializer, CoinTransactionSerializer, 
-    SurveyResponseSerializer, VideoSerializer, EmotionSerializer, ShareLinkSerializer, AnalyticsReportSerializer,
-    VideoDetailSerializer, CompanyDetailSerializer, CustomUserDetailSerializer
-)
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from firebase_admin import auth
+from django.contrib.auth import login
+from .models import ViewerProfile, User
+from .serializers import OnboardingSerializer
+from django.http import HttpResponseRedirect
```

Ruff (0.8.2)

`django.http.HttpResponseRedirect` imported but unused

Remove unused import: `django.http.HttpResponseRedirect`

(F401)